site/content/en/docs/drivers/_index.md

```diff
@@ -31,6 +31,16 @@ To do so, we use the [Docker Machine](https://github.com/docker/machine) library
 * [VMware Fusion]({{<ref "vmware.md">}}) - VM
 * [SSH]({{<ref "ssh.md">}}) - remote ssh
 
+### Apple Silicon Support
+
+The following drivers support Apple Silicon users:
+* Docker
+* Parallels - Requires an ISO that supports arm64
```

Comment:
currently we dont have arm64 ISO yet, so we can remove these

Reply:
Apologies, I misinterpreted the comment below from the issue. Will remove.

> Both vmware and parallels are supposed to work, but you also need an ISO that supports arm64.

Reply:
Removed.

Reply:
Once there is an ISO, it will be easier to test if the minikube driver actually works with it.

But both parallels and vmware **do** have support for arm64 on Mac, that was what I meant.